src/main/java/org/openstreetmap/atlas/checks/base/CheckResourceLoader.java

```diff
@@ -83,29 +96,59 @@ public CheckResourceLoader(final Configuration configuration)
         this.configuration = configuration;
     }
 
+    public Configuration getConfiguration()
+    {
+        return this.configuration;
+    }
+
     /**
-     * Loads checks that are enabled by configuration
+     * Get configuration for a specific country, overriding for country specific overrides and group
+     * specific overrides
      *
-     * @param <T>
-     *            check type
-     * @return a {@link Set} of checks
+     * @param country
+     *            country string
+     * @return {@link Configuration}
      */
-    public <T extends Check> Set<T> loadChecks()
+    public Configuration getConfigurationForCountry(final String country)
     {
-        return loadChecks(this::isEnabledByConfiguration);
+        Configuration specializedConfiguration = this.configuration
+                .configurationForKeyword(country);
+
+        if (this.countryGroups.containsKey(country))
+        {
+            for (final String group : this.countryGroups.get(country))
```

Comment:
Although hash table is small here, you are doing lookup twice if given country exists in the map: one with `containsKey` and another with `get` call. I'd suggest using `get` and checking against null values.